airbyte-webapp/src/components/connection/CatalogTree/next/DestinationNamespaceModal.tsx

```diff
@@ -0,0 +1,346 @@
+              }),
+            },
+            {
+              id: "destinationNamespace",
+              displayName: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.header.destinationNamespace",
+              }),
+            },
+          ],
+          data: [
+            {
+              customFormat: formatMessage(
+                {
+                  id: "connectionForm.modal.destinationNamespace.table.data.custom",
+                },
+                { symbol: (node: React.ReactNode) => `"${node}"` }
+              ),
+              sourceNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.public",
+              }),
+              destinationNamespace: formatMessage(
+                {
+                  id: "connectionForm.modal.destinationNamespace.table.data.custom",
+                },
+                { symbol: (node: React.ReactNode) => `${node}` }
+              ),
+            },
+            {
+              customFormat: formatMessage(
+                {
+                  id: "connectionForm.modal.destinationNamespace.table.data.exampleSourceNamespace",
+                },
+                { symbol: (node: React.ReactNode) => `{${node}}` }
+              ),
+              sourceNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.public",
+              }),
+              destinationNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.public",
+              }),
+            },
+            {
+              customFormat: formatMessage(
+                {
+                  id: "connectionForm.modal.destinationNamespace.table.data.exampleMySourceNamespace",
+                },
+                { symbol: (node: React.ReactNode) => `{${node}}` }
+              ),
+              sourceNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.public",
+              }),
+              destinationNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.myPublicSchema",
+              }),
+            },
+            {
+              customFormat: "",
+              sourceNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.public",
+              }),
+              destinationNamespace: formatMessage({
+                id: "connectionForm.modal.destinationNamespace.table.data.mySchema",
+              }),
+            },
+          ],
+        };
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={{
+        namespaceDefinition: initialValues?.namespaceDefinition ?? NamespaceDefinitionType.source,
+        namespaceFormat: initialValues.namespaceFormat,
+      }}
+      enableReinitialize
+      validateOnBlur
+      validateOnChange={false}
```

Comment:
Any reason why we don't wanna validate on change?) 😃 
Since I noticed even after we've entered the valid input, the forms remain invalid, and only after the `onBlur` event occurs, the form become valid.
IMO we need to give the user faster feedback, don't make him think about what to do next.
The same question for 
airbyte-webapp/src/components/connection/CatalogTree/next/DestinationStreamNamesModal.tsx

Reply:
Good idea. Thanks :slightly_smiling_face:

Reply:
Verified - fixed